### PR TITLE
Chronological history view

### PR DESCRIPTION
**Pull request by tinoweiss**
Adds a history view (visible from home page) that displays record history for all assets for the past 3 months in reverse chronological order.

### CODE REVIEW

File: client/src/app/views/history/history.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            }
+            .record-name {
+                width: 50%;
+                font-size: 16px;
+                padding-bottom: 5px;
+                float: right;
+                text-align: right;
+            }
+            .checked-out-by {
+                color: $checkit-subcolor
```
Comment (TimCreasman) [inline client/src/app/views/history/history.scss]: Add a semi-colon at the end

File: client/src/app/views/history/history.route.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		$urlRouterProvider.otherwise('/');
+		$stateProvider
+            .state('history', {
+                parent: 'root',
+                url: '/history',
+                views: {
+                    'content@': {
+                        templateUrl: 'app/views/history/history.html',
+                        controller: 'HistoryController',
+						controllerAs: 'vm'
```
Comment (TimCreasman) [inline client/src/app/views/history/history.route.js]: Fix indentation

File: client/src/app/views/list/list.controller.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
  * This file holds the definition for the list controller for the list page.
```
Comment (TimCreasman) [inline client/src/app/views/list/list.controller.js]: Should this file be a separate commit? How does it relate to the history view?

Comment (tinoweiss) [inline client/src/app/views/list/list.controller.js]: I think the autoformat in my editor just updated some spacing/indentation in this file. Nothing actually changed functionality-wise.
